### PR TITLE
Resolve #107 Show Shapefile Download Icon

### PR DESCRIPTION
**Pull request by mzagaja**
Show shapefile download button for the appropriate kinds of datasets. Now we will stop showing them for tabular datasets and show it for geospatial datasets.

### CODE REVIEW

File: app/templates/datasets.hbs [hunk 1]
```diff
@@ -54,14 +54,11 @@
...
             </a>
 
-            {{#if model.raw_data.spatialMetaData}}
+            {{#unless model.raw_data.spatialMetaData}}
               <a class="button lift" href={{download_link_shapefile}} download={{model.dataset.table_name}}>
                 .shp
               </a>
-              <a class="button lift" href={{download_link_geojson}} download={{model.dataset.table_name}}>
-                .geojson
-              </a>
```
Comment (ericyoungberg) [inline app/templates/datasets.hbs:63]: This PR removes GeoJSON support?

File: app/templates/datasets.hbs [hunk 2]
```diff
@@ -54,14 +54,11 @@
               .csv
             </a>
 
-            {{#if model.raw_data.spatialMetaData}}
+            {{#unless model.raw_data.spatialMetaData}}
```
Comment (ericyoungberg) [inline app/templates/datasets.hbs]: This will make the spatial button show up only if there isn't spatial metadata

**Review by ericyoungberg — CHANGES_REQUESTED** [review]

File: app/templates/datasets.hbs (hunk 1 above)
Comment (mzagaja) [reply app/templates/datasets.hbs:63]: Not specifically but GeoJSON is dependent upon CartoDB so we would need to build GeoJSON endpoints as a new Data common feature when we pull out Carto. I can leave it there but at some point we should remove it from the UI until it is rebuilt.

File: app/templates/datasets.hbs (hunk 2 above)
Comment (mzagaja) [reply app/templates/datasets.hbs]: So this particular piece of code is a bit confusing. What it is looking for the spatialMetadata about tabular data, not the spatialMetadata about spatial data. We can refactor it to name this something else, but if you test this on staging or in development you should see that it properly hides the shapefile download for tabular data and shows it for geospatial.

Comment (ericyoungberg) [reply app/templates/datasets.hbs]:
```js
model.dataset.get('schemaname')
```

**Review by ericyoungberg — APPROVED** [review]

**Pull request merged**